unit_tests/unit_test_rules.mk

```diff
@@ -63,7 +70,11 @@ endif
 
 # Enable address sanitizer for C++ files, but not on Windows since x86_64-w64-mingw32-g++ doesn't support it.
 ifeq ($(SANITIZE),yes)
-	USE_CPPOPT += -fsanitize=address -fsanitize=bounds-strict -fno-sanitize-recover=all
+	ifeq ($(IS_MAC),yes)
+		USE_CPPOPT += -fsanitize=address
+	else
+		USE_CPPOPT += -fsanitize=address -fsanitize=bounds-strict -fno-sanitize-recover=all
```

Comment:
why only C++ files?  Should be all, right?

Reply:
only c++ because lua does some things asan doesn't like, but don't actually cause overruns.

Reply:
@mck1117 that would have been an amazing code comment ^^^